stage0/.cargo/config.toml

```diff
@@ -2,9 +2,9 @@
 target = "x86_64-unknown-none"
 
 [target.x86_64-unknown-none]
-rustflags = "-C relocation-model=static -C code-model=large"
+rustflags = "-C relocation-model=static -C code-model=large -C linker-plugin-lto"
 
 [unstable]
 # We need to build std / core because of the code model. This may be removed if / when it's possible to build stage0 without it in the future.
-build-std = ["core"]
+build-std = ["core", "alloc"]
```

Comment:
We managed to get so far without this :(

Reply:
Indeed.